db/migrate/20170326022716_add_article_image_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class AddArticleImageToArticles < ActiveRecord::Migration[5.0]
+  def change
+    add_column :articles, :article_image, :string
```

Comment:
so as not to clobber the current column.

it might be good to do some sort of migration and then rename this back to just `image`